Move to .NET 6.0

terrajobst opened the pull request:
- Upgrade to .NET Framework 4.72
- Upgrade dependencies
- Address xUnit warnings
- Move to .NET 6.0

dallmair commented:
LGTM 😉 

Except for maybe `src/NQueryViewer/App.config`, which still lists 4.7.2 as `supportedRuntime` although the project targets `net6.0-windows` -- remove the file?

Also, the commented out lines in `src/NQueryViewer/Properties/AssemblyInfo.cs` looks a bit strange and probably should just be removed.

terrajobst commented:
> Except for maybe `src/NQueryViewer/App.config`, which still lists 4.7.2 as `supportedRuntime` although the project targets `net6.0-windows` -- remove the file?

Good call. I've also removed the others which were used to configure xUnit tests. I've switched migrated this to `xunit.runner.json` and added that to the shared targets.
